Classes/Controller/AddRedirectsController.php

```diff
@@ -192,7 +211,10 @@ protected function processRedirects(array $redirectListArray): bool
         // precessing each line in the array
         foreach ($redirectListArray as $item){
             // separate the row columns
-            $row = explode($this->separatedChars[$this->selectedSeparatedChar],$item);
+            $row = GeneralUtility::trimExplode($this->separatedChars[$this->selectedSeparatedChar],$item);
```

Comment:
ne pas laisser des instructions trop longues : 
$separator = $this->separators[$this->selectedSeparator];
$row = explode($separator, ...];